games/d.yaml

```diff
@@ -192,6 +192,20 @@
     - http://clintbellanger.net/rpg/images/screenshots/0020.jpg
     - http://clintbellanger.net/rpg/images/screenshots/0022.jpg
 
+- name: [Digger, Digger (video game)]
+  meta:
+    genre: [Action]
+  remakes:
+  - name: Digger Remastered
+    url: http://digger.org/
+    repo: http://digger.org/download.html
+    status: playable
+    development: halted
+    lang: C
+    license: [GPL]
+    images:
+    - https://en.wikipedia.org/wiki/Digger_(video_game)#/media/File:Digger_Remastered.png
```

Comment:
Can you put `https://upload.wikimedia.org/wikipedia/commons/b/be/Digger_Remastered.png` there? It's better to have link to an actual image here, since we're embedding them on the site.